src/components/ButtonAppBar/index.js

```diff
@@ -42,29 +42,35 @@ const useStyles = makeStyles(theme => ({
   },
 }))
 
-const divStyle = {
-  display: 'inline',
-}
+const divStyle = { display: 'inline' }
 
 export function ButtonAppBar() {
+  const classes = useStyles()
+  const history = useHistory()
+  const pageHistory = useHistory()
   const { userInfo, logout } = useUserContext()
 
+  const handleTitle = event => {
+    pageHistory.push('/ItemList')
+    history.push('/ItemList')
+  }
+
   const signOut = () => {
     logout()
   }
-
+  
   const LinkItems = React.forwardRef((props, ref) => (
     <RouterLink innerRef={ref} to={routes.ITEMS} {...props} />
   ))
-
-  const classes = useStyles()
-
+  
   return (
     <div className={classes.root}>
       <AppBar position="static">
         <Toolbar>
           <Typography variant="h6" className={classes.title}>
             <Link
+              to="/ItemList"
```

Comment:
It needs to be make sure if this `<Link>` component is one of Material UI or one of React Router. If it is one of Material UI, I don't think we want to have `to` property here.